packages/firestore/test/integration/api/transactions.test.ts

```diff
@@ -800,6 +805,28 @@ apiDescribe('Database transactions', (persistence: boolean) => {
     });
   });
 
+  it('does not retry on permanent errors', () => {
+    return integrationHelpers.withTestDb(persistence, db => {
+      let count = 0;
+      return db
```

Comment:
On android you had a "Make a transaction that should fail with a permanent error" comment...